README.md

```diff
@@ -1,3 +1,190 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [My Portfolio] <a name="Basic HTML and css Page"></a>
+
+> This is basic page I build it with HTML and CSS
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. Linters
+2. HTML
+3. CSS
+
+### Tech Stack <a name="tech-stack"></a>
+1. Linters
+2. HTML
+3. CSS
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss.org/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+1. Added Linters.yml file.
+2. Added index.html & style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-Demo will be added soon. 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser <br/>
+Code editor
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:<br/>
+git clone git@github.com:bhushan354/Portfolio_Project.git
+
+
+### Install
+
+Install this project with these commands:
+
+Install live server extension in VS code then open index.html file using live server.
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Install live server extension in VS code then open index.html file using live server.
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming soon.
+
+
+
+### Deployment
+
+Coming soon.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+Bhushan Deshmukh
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/bhushan354)
+- Twitter: [@twitterhandle](https://twitter.com/Bhushan_4885)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] Add my new projects
+- [ ] make it ready for deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Do follow this account if you liked this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and my peers for providing enough knowledge for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+
```

Comment:
- [ ] I noticed the  MIT link here is broken as it's redirecting to a "404 page not found" the reason for this is that you don't have a license file in your project, so please make sure to add a license file and the MIT link should redirect to that file.
- You can find an example [license file](https://github.com/microverseinc/readme-template/blob/master/MIT.md) here, just make sure to update the content inside the file. 😉